### ISSUE TITLE
Cannot collapse the sidebar in viewer or when the editor is hidden

### ISSUE THREAD

**Issue by jasonLaster** [RecordReplay/devtools#3746]
When the editor is collapsed the sidebar does not collapse on double click

https://app.replay.io/recording/3c789ad1-11b1-4de9-a97f-aba6f2b049a5

by the way, i'm second guessing our decision to set a fixed width on the left sidebar... the reason is sometimes it feels like the pause panel should be wider....

**Comment by jasonLaster** [comment]
Related to the first bug

**Comment by jcmorrow** [comment]
Yep, I know why this is happening, should be resolved by my fix for https://github.com/RecordReplay/devtools/issues/3851 once I have it finalized.
